src/cdk-experimental/column-resize/overlay-handle.ts

```diff
@@ -0,0 +1,207 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {AfterViewInit, ElementRef, OnDestroy, NgZone} from '@angular/core';
+import {coerceCssPixelValue} from '@angular/cdk/coercion';
+import {Directionality} from '@angular/cdk/bidi';
+import {ESCAPE} from '@angular/cdk/keycodes';
+import {CdkColumnDef} from '@angular/cdk/table';
+import {fromEvent, ReplaySubject} from 'rxjs';
+import {
+  distinctUntilChanged,
+  filter,
+  map,
+  mapTo,
+  pairwise,
+  startWith,
+  takeUntil,
+} from 'rxjs/operators';
+
+import {_closest} from '@angular/cdk-experimental/popover-edit';
+
+import {HEADER_CELL_SELECTOR} from './selectors';
+import {ColumnResizeNotifierSource} from './column-resize-notifier';
+import {HeaderRowEventDispatcher} from './event-dispatcher';
+import {ResizeRef} from './resize-ref';
+
+// TODO: Take another look at using cdk drag drop. IIRC I ran into a couple
+// good reasons for not using it but I don't remember what they were at this point.
+/**
+ * Base class for a component shown over the edge of a resizable column that is responsible
+ * for handling column resize mouse events and displaying any visible UI on the column edge.
+ */
+export abstract class ResizeOverlayHandle implements AfterViewInit, OnDestroy {
+  protected readonly destroyed = new ReplaySubject<void>();
+
+  protected abstract readonly columnDef: CdkColumnDef;
+  protected abstract readonly document: Document;
+  protected abstract readonly directionality: Directionality;
+  protected abstract readonly elementRef: ElementRef;
+  protected abstract readonly eventDispatcher: HeaderRowEventDispatcher;
+  protected abstract readonly ngZone: NgZone;
+  protected abstract readonly resizeNotifier: ColumnResizeNotifierSource;
+  protected abstract readonly resizeRef: ResizeRef;
+
+  ngAfterViewInit() {
+    this._listenForMouseEvents();
+  }
+
+  ngOnDestroy() {
+    this.destroyed.next();
+    this.destroyed.complete();
+  }
+
+  private _listenForMouseEvents() {
+    this.ngZone.runOutsideAngular(() => {
+      const takeUntilDestroyed = takeUntil<MouseEvent>(this.destroyed);
+
+      fromEvent<MouseEvent>(this.elementRef.nativeElement!, 'mouseenter').pipe(
+          takeUntilDestroyed,
+          mapTo(this.resizeRef.origin.nativeElement!),
+      ).subscribe(cell => this.eventDispatcher.headerCellHovered.next(cell));
+
+      fromEvent<MouseEvent>(this.elementRef.nativeElement!, 'mouseleave').pipe(
+          takeUntilDestroyed,
+          map(event => event.relatedTarget &&
```

Comment:
Have you tested this out in different browsers? I know that IE at least used to have some issues with `relatedTarget`.

Reply:
I don't have a good way to test it on IE (I just have access to Mac, Linux, ChomeOS and iOS). I'd appreciate help sorting things like this out before this graduates from experimental.

Reply:
Windows, IE11, and Edge (non-Chromium) run fine for me on my MBPs via VirtualBox, VMWare Fusion, and Bootcamp. VirtualBox and Bootcamp are free.